### PR TITLE
Fix transport tcp failure in gluster disk

### PR DESCRIPTION
**Pull request by chunfuwen**
Need wrap transport attribute into host instead of standalone attribute

### CODE REVIEW

**Comment by chunfuwen** [comment]
@will-Do 
@waynesun09 
@ lento-sun

Please help take a look.

**Comment by chunfuwen** [comment]
@lento-sun

File: libvirt/tests/src/virtual_disks/virtual_disks_gluster.py [hunk 1]
```diff
@@ -89,13 +89,16 @@ def build_disk_xml(disk_img, disk_format, host_ip):
         else:
             source_dict = {"protocol": "gluster",
                            "name": "%s/%s" % (vol_name, disk_img)}
-            host_dict = [{"name": host_ip, "port": "24007"}]
+            host_dict = []
+            # If transport is True, add attribute transport to it
+            if transport:
+                host_dict.append({"name": host_ip, "transport": transport, "port": "24007"})
+            else:
+                host_dict.append({"name": host_ip, "port": "24007"})
```
Comment (will-Do) [inline libvirt/tests/src/virtual_disks/virtual_disks_gluster.py]: How about just update the first dict of the list:
```
host_dict = [{"name": host_ip, "port": "24007"}]
if transport:
    host_dict[0]['transport'] = transport

**Comment by chunfuwen** [comment]
@will-Do,

Update script to host_dict[0]['transport''] = transport

**Pull request merged**